pages/_app.tsx

```diff
@@ -20,6 +21,7 @@ export default function App({ Component, pageProps }: AppProps) {
         className={`${inter.className} bg-white dark:bg-black flex flex-col items-center justify-start px-10 min-h-screen"`}
       >
         <Component {...pageProps} />
+        <Graph></Graph>
```

Comment:
Can this be self closing?
```suggestion
        <Graph />
```